tests/server/server-executeBulkWrite-002.phpt

```diff
@@ -0,0 +1,34 @@
+--TEST--
+MongoDB\Driver\Server::executeBulkWrite() takes a write concern (standalone)
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; CLEANUP(STANDALONE) ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(STANDALONE);
+$primary = $manager->selectServer(new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_PRIMARY));
```

Comment:
Since someone will inevitably do it, can you add a test where you have $secondary->executeBulkWrite("local.collection", $bulk, $writeConcern) ?

And also $secondary->executeBulkWrite(NS, $bulk, $writeConcern); just for completeness